src/lang-promql/complete/hybrid.ts

```diff
@@ -42,20 +42,21 @@ import {
   MetricIdentifier,
   StringLiteral,
   VectorSelector,
+  AggregateModifier,
 } from 'lezer-promql';
 
-const matchOp = 9999;
-
 interface AutoCompleteNode {
   labels: string[];
   type: string;
 }
 
-const autocompleteNode: { [key: number]: AutoCompleteNode } = {
-  [matchOp]: { labels: matchOpTerms, type: '' },
-  [BinaryExpr]: { labels: binOpTerms, type: '' },
-  [FunctionIdentifier]: { labels: functionIdentifierTerms, type: 'function' },
-  [AggregateOp]: { labels: aggregateOpTerms, type: 'keyword' },
+const autocompleteNode: { [key: string]: AutoCompleteNode } = {
+  matchOp: { labels: matchOpTerms, type: '' },
+  binOp: { labels: binOpTerms, type: '' },
+  binOpModifier: { labels: ['on', 'ignoring', 'group_left', 'group_right'], type: 'keyword' },
```

Comment:
you don't want to add this list to lezer-promql ? It would be consistent like that

Reply:
Hm ok yeah, I added these to lezer and updated lezer here. I'm still really not happy about those things living there, but I don't know a better solution either for the moment.